backend/data.py

```diff
@@ -0,0 +1,84 @@
+from flask import request
+from flask import jsonify
+from flask import current_app
+from backend.db import *
+
+app = current_app
+
+@app.route("/getcommit", methods=("GET","POST"))
```

Comment:
Nit: This looks more like "/appendcommit" than "/getcommit" to me. Wonder what GET method do in this.

Reply:
I haven't tested the function yet (I' ll change a better index lol). Just added two methods, maybe deleting GET after testing.

Reply:
NP. Looks good to me. Thank you for the good work!